constants.scad

```diff
@@ -13,7 +13,7 @@ servo_screw_offsets=[3, 5];
 servo_screw_size = 4;
 servo_gear_teeth_size = 1;
 servo_gear_num_teeth = 12;
-servo_gear_thickness = 5;
+servo_gear_thickness = 6;
```

Comment:
I see now why this is thicker...just FYI, we may not be able to make it thicker. This in theory depends on the gear that comes with our servos, unless we like maybe design a slip on extender? We can get to that if it becomes an issue though!

Reply:
Ah, I think we'll need an adapter either way.  Servos often come with "propellers" which we can affix the gear to.